profile_library/calex/G95 Gold/model.json

```
@@ -0,0 +1,17 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-04-28T23:34:57.359526",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plus Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.11:docker"
+  },
+  "name": "G95Gold",
```

name prop might have some more extensive information / contain the fully marketed name of the product.

```suggestion
  "name": "Smart LED Filament Gold Globe-lamp G95 7W",
```

---

Got the name from Home Assistant didn't know if auto-discovery would work if I had use the full name

---

Yes. More information about Powercalc auto discovery routine can be found here: https://docs.powercalc.nl/library/discovery/
Name is only used on the library website: https://library.powercalc.nl